restaurant/src/app/menu-page/category-meals/category-meals.component.html

```diff
@@ -1,142 +1,180 @@
 <!-- <div  class="test">
   <p>{{ newMealInfo.mealName || null }}</p>
 </div> -->
+  <div class="main-wraper">
+    <!-- GRID START -->
 
-<div class="main-wraper">
-  <!-- GRID START -->
-  <div class="grid-wraper">
-    <!-- ADD NEW MEAL BUTTON START -->
-    <div class="add-new-meal-div">
-      <button (click)="goToNewMealPage()" class="button">Add new meal</button>
-    </div>
-    <!-- ADD NEW MEAL BUTTON END -->
-    <!-- SEARCH START -->
-    <form class="search-form-div" [formGroup]="searchForm">
-      <input
-        placeholder="Search food here"
-        type="text"
-        formControlName="search"
-        (keyup)="searchValueChange($event)"
-      />
-    </form>
-    <!-- SEARCH END -->
-    <div class="grid-choices">
-      <ul
-        *ngFor="
-          let categoryMeal of categoryMeals
-            | slice
-              : (defaultPage - 1) * itemsPerPage
-              : defaultPage * itemsPerPage
-        "
-      >
-        <li (click)="getMealDetails(categoryMeal.idMeal)">
-          <div class="meal-list-name">
-            <span> {{ categoryMeal.strMeal }}</span>
-          </div>
-        </li>
-      </ul>
-      <!-- PAGINATION START -->
-      <div *ngIf="!isSearching" class="pag-div">
-        <ngb-pagination
-          [collectionSize]="numberOfMeals"
-          [(page)]="defaultPage"
-          [directionLinks]="false"
-        ></ngb-pagination>
+    <div class="grid-wraper">
+      <!-- ADD NEW MEAL BUTTON START -->
+      <div class="add-new-meal-div">
+        <button (click)="goToNewMealPage()" class="button">Add new meal</button>
+      </div>
+      <!-- ADD NEW MEAL BUTTON END -->
+      <!-- SEARCH START -->
+      <form class="search-form-div" [formGroup]="searchForm">
+        <input
+          placeholder="Search food here"
+          type="text"
+          formControlName="search"
+          (keyup)="searchValueChange($event)"
```

Comment:
ima li metoda onchange? keyup nece da radi na copy paste, tj on reaguje na input, a onchange ce da reaguje na svaku promenu. Mada u formama angularovim se radi na subscribe tipa  this.myForm.get('myControl')?.valueChanges.subscribe ili tako nesto